app/controllers/sections/items/ItemBulkPackagingSealChoiceController.scala

```diff
@@ -48,7 +48,8 @@ class ItemBulkPackagingSealChoiceController @Inject()(
     authorisedDataRequestAsync(ern, draftId) { implicit request =>
       validateIndexAsync(idx) {
         withItemBulkPackaging(idx) { description =>
-          renderView(Ok, fillForm(ItemBulkPackagingSealChoicePage(idx), formProvider()), idx, mode, description)
+          val isBulk = request.userAnswers.get(ItemBulkPackagingChoicePage(idx)).get
```

Comment:
We already know that this is the bulk flow as there are 2 controllers, one for individual packaging and one for bulk. This can be removed.